conda/meta.yaml

```diff
@@ -1,22 +1,27 @@
+{% set name = "zstash" %}
+{% set version = "0.4.0" %}
+
 package:
-    name: zstash
-    version: 0.4.0
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    git_url: https://github.com/E3SM-Project/zstash
-    git_rev: v0.4.0
+  git_url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 9fce7561efa0d512567f89219a1f1642210810523a550b1974a17fdba1b6f544
 
 build:
-    script: python setup.py install
+  number: 1
```

Comment:
Very important to have a build number.  I set it to 1 because build 0 has already been uploaded.  Never overwrite a build unless you have a very good reason to do so.